sound/soc/sof/intel/hda-stream.c

```diff
@@ -279,6 +279,41 @@ int hda_dsp_stream_put(struct snd_sof_dev *sdev, int direction, int stream_tag)
 	return 0;
 }
 
+int hda_dsp_stream_reset(struct snd_sof_dev *sdev, struct hdac_stream *hstream)
```

Comment:
On commit message, "This would mean that we leave them decoupled during system suspend." Should this sentence be more specific to when the pcm is active?